gravitee-apim-console-webui/src/services-ngx/notification-settings.service.ts

```diff
@@ -33,4 +33,12 @@ export class NotificationSettingsService {
   delete(apiId: string, id: string): Observable<NotificationSettings[]> {
     return this.http.delete<NotificationSettings[]>(`${this.constants.env.baseURL}/apis/${apiId}/notificationsettings/${id}`);
   }
+
+  getNotifiers(apiId: string): Observable<any> {
```

Comment:
WDYT to type the return of this function  and type the get<...>(..)?
And same add test on it

Reply:
Added. But not sure where to add it test file since I'm not actually using this method there

Reply:
Okay, added